src/js/components/molecules/article-table/data/article-table.data.ts

```diff
@@ -0,0 +1,51 @@
+/** @format */
```

Comment:
i prefer splitting up the examples and placing them an example folder under data and then pull them in; this illustrates what i mean: `src/js/components/organisms/overview/data/overview.data.ts`